woocommerce-gateway-ebanx/includes/class-wc-ebanx-global-gateway.php

```diff
@@ -53,10 +53,15 @@
 	 */
 	public function __construct()
 	{
+		include_once(INCLUDES_DIR . 'notices/class-wc-ebanx-notices-notice.php');
```

Comment:
Myabe can we remove include_once from here and insert out of the class. Maybe on top of the file? What do you think?